frontend/test/metabase/scenarios/admin/datamodel/reproductions/21984-data-model-registered-as-view.cy.spec.js

```diff
@@ -0,0 +1,36 @@
+import { restore } from "__support__/e2e/helpers";
+
+import { SAMPLE_DB_ID } from "__support__/e2e/cypress_data";
+import { SAMPLE_DATABASE } from "__support__/e2e/cypress_sample_database";
+
+const { REVIEWS_ID } = SAMPLE_DATABASE;
+
+const reviewsDataModelPage = `/admin/datamodel/database/${SAMPLE_DB_ID}/table/${REVIEWS_ID}`;
+
+describe.skip("issue 21984", () => {
+  beforeEach(() => {
+    cy.intercept("GET", "/api/table/*/query_metadata?**").as("tableMetadata");
+
+    restore();
+    cy.signInAsAdmin();
+
+    cy.visit(reviewsDataModelPage);
+    cy.wait(["@tableMetadata", "@tableMetadata"]);
+
+    cy.findByDisplayValue("ID");
+  });
+
+  it('should not show data model visited tables in search or in "Pick up where you left off" items on homepage (metabase#21984)', () => {
+    cy.visit("/");
+
+    cy.findByText("Pick up where you left off")
```

Comment:
Don't you need to wait for `/api/activity/recent_views` here?

Reply:
It worked every single time I ran this test and I did it at least 20 times locally. It should be enough to rely on UI in this case.